squid-blockexplorer/.gitignore

```diff
@@ -0,0 +1,8 @@
+/node_modules
+/lib
+
+/**Versions.json
+/**Versions.jsonl
+
+# IDE files
+/.idea
```

Comment:
nitpick: Maybe this should be [in a local `gitignore`](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)?